records_mover/db/driver.py

```diff
@@ -134,6 +116,49 @@ def load_from_fileobj(self, schema: str, table: str,
         """
         raise NotImplementedError(f"load_from_fileobj not implemented for this database type")
 
+
+
+
+
+
+
+class DBDriver(LoadsFromRecordsDirectory,
+               NegotiatesLoadFormat,
+               UnloadsToRecordsDirectory,
+               LoadsFromFileobj):
+    def __init__(self,
+                 db: Union[sqlalchemy.engine.Engine,
+                           sqlalchemy.engine.Connection], **kwargs) -> None:
+        self.db = db
+        self.db_engine = db.engine
+        self.meta = MetaData()
+
+    def has_table(self, schema: str, table: str) -> bool:
+        return self.db.dialect.has_table(self.db, table_name=table, schema=schema)
+
+    def table(self,
+              schema: str,
+              table: str) -> Table:
+        return Table(table, self.meta, schema=schema, autoload=True, autoload_with=self.db_engine)
+
+    def schema_sql(self,
+                   schema: str,
+                   table: str) -> str:
+        """Generate DDL which will recreate the specified table and return it
+        as a string.  Returns None if database or permissions don't
+        support operation.
+
+        """
+        # http://docs.sqlalchemy.org/en/latest/core/reflection.html
+        table_obj = self.table(schema, table)
+        return str(CreateTable(table_obj, bind=self.db))
+
+    # TODO: Is this for load or unload?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker